manifests/config_aem_ssl.pp

```diff
@@ -0,0 +1,87 @@
+define aem_curator::config_aem_ssl (
+  $tmp_dir,
+  $run_mode,
+  $aem_ssl_port,
+  $host,
+  $aem_base                      = '/opt',
+  $aem_id                        = 'aem',
+  $aem_ssl_method                = 'jetty',
+  $aem_keystore_password         = undef,
+  $aem_keystore_path             = undef,
+  $cert_base_url                 = undef,
+) {
+  $keystore_path = pick(
+    $aem_keystore_path,
+    "${aem_base}/aem/${aem_id}/crx-quickstart/ssl/aem.ks",
+  )
+  file { dirname($keystore_path):
+    ensure => directory,
+    mode   => '0770',
+    owner  => "aem-${aem_id}",
+    group  => "aem-${aem_id}",
+  }
+  if !defined(File[$tmp_dir]) {
+    file { $tmp_dir:
+      ensure => directory,
+    }
+  }
+  if !defined(File["${tmp_dir}/${aem_id}"]) {
+    file { "${tmp_dir}/${aem_id}":
+      ensure => directory,
+      mode   => '0700',
+    }
+  }
+
```

Comment:
Are these lines required for enabling ssl via granite & jetty ?

Reply:
Yes they are